app/src/main/res/values-ko/strings.xml

```diff
@@ -0,0 +1,10 @@
+<resources>
+    <string name="app_name">TedPermissionDemo</string>
+
+
+    <string name="custom_close_text">Custom Close</string>
```

Comment:
Please translate to korean language for `values-ko/strings.xml`
Please modify other things as well.

Reply:
You're right!
I'll push changes. Thanks!